### PR TITLE
PP-12735 Change RESTClientError import to pay-js-commons class

### PR DESCRIPTION
**Pull request by hjvoid**
## WHAT
Updating the post-configure.controller to use the RESTClientError from pay-js-commons/axios-client so that correct error message is shown to users as opposed to a 500 internal server error message.

## HOW 
_Steps to test or reproduce:_

1. Sign in to an existing user account in the admin tool
2. Follow the steps to set up a new authenticator app until you reach the page where you have to enter the code from the app.
3. Deliberately enter an incorrect code.
4. You should now see the following error message:
> The security code you entered is not correct, try entering it again or wait for your authenticator app to give you a new code
Screenshot 2024-07-11 at 17 02 15
Screenshot 2024-07-11 at 17 02 26

_Note: There are other instances of the class that was causing this bug which are being addressed in another ticket_ [here](https://payments-platform.atlassian.net/browse/PP-12868).

### CODE REVIEW

File: app/controllers/user/two-factor-auth/post-configure.controller.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const lodash = require('lodash')
 const paths = require('../../../paths')
 const userService = require('../../../services/user.service.js')
 const secondFactorMethod = require('../../../models/second-factor-method')
-const { RESTClientError } = require('../../../errors')
+const { RESTClientError } = require('@govuk-pay/pay-js-commons/lib/utils/axios-base-client/errors.js')
```
Comment (iqbalgds) [inline app/controllers/user/two-factor-auth/post-configure.controller.js:8]: Can we please get rid of the .js file extension - thanks

File: app/controllers/user/two-factor-auth/post-configure.controller.test.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const { expect } = require('chai')
 const userFixtures = require('../../../../test/fixtures/user.fixtures')
 const User = require('../../../models/User.class')
 const paths = require('../../../paths')
-const { RESTClientError } = require('../../../errors')
+const { RESTClientError } = require('@govuk-pay/pay-js-commons/lib/utils/axios-base-client/errors.js')
```
Comment (iqbalgds) [inline app/controllers/user/two-factor-auth/post-configure.controller.test.js:8]: Same again - thanks!

**Review by iqbalgds — CHANGES_REQUESTED** [review]
Looks great - tiny teeny change requested

File: app/controllers/user/two-factor-auth/post-configure.controller.js (hunk 1 above)
Comment (hjvoid) [reply app/controllers/user/two-factor-auth/post-configure.controller.js]: Done.

**Review by alexbishop1 — APPROVED** [review]

**Review by iqbalgds — APPROVED** [review]

**Pull request merged**